src/oc/auth/app.clj

```diff
@@ -57,6 +58,7 @@
     "Database: " c/db-name "\n"
     "Database pool: " c/db-pool-size "\n"
     "AWS SQS email queue: " c/aws-sqs-email-queue "\n"
+    "AWS SQS email queue: " c/aws-sqs-slack-router-auth-queue "\n"
```

Comment:
@thepug  You print email queue 2 times, the second should be slack queue or something else, right?

Reply:
cut n paste for the win